iac-extensions/docker/src/test/resources/checks/RetrieveRemoteResourcesCheck/curl.dockerfile

```diff
@@ -428,7 +398,29 @@ RUN curl -L https://example.com/resource -s --output output.txt -k -H "X-Auth-To
 RUN curl --header "X-Auth-Token 123" https://example.com/resource --output output.txt -L
 RUN curl -L https://example.com/resource -s --output output.txt --header "X-Auth-Token 123" -k
 
-# Compliant save in /dev/null
+# Compliant: ADD doesn’t support HTTP request options =======
+
+RUN curl -o output.txt https://example.com/resource --data "param1=value1"
+RUN curl -o output.txt https://example.com/resource -d "param1=value1"
+RUN curl -o output.txt https://example.com/resource --data-ascii @file
+RUN curl -o output.txt https://example.com/resource --data-binary @file
+RUN curl -o output.txt https://example.com/resource --data-raw "param1=value1"
+RUN curl -o output.txt https://example.com/resource --data-urlencode param1=value1
+RUN curl -o output.txt https://example.com/resource --form param1=value1
+RUN curl -o output.txt https://example.com/resource -F param1=value1
+RUN curl -o output.txt https://example.com/resource --form-escape -F 'param1=value1'
+RUN curl -o output.txt https://example.com/resource --form-string "param1=value1"
+RUN curl -o output.txt https://example.com/resource --header "Content-Type: application/json"
+RUN curl -o output.txt https://example.com/resource -H "Content-Type: application/json"
+RUN curl -o output.txt https://example.com/resource --json '{"param1": "value1"}'
+RUN curl -o output.txt https://example.com/resource --referer "https://example.com/referer"
+RUN curl -o output.txt https://example.com/resource -e "https://example.com/referer"
+RUN curl -o output.txt https://example.com/resource --request POST
+RUN curl -o output.txt https://example.com/resource -X POST
+RUN curl -o output.txt https://example.com/resource --user-agent "Mozilla/5.0"
+RUN curl -o output.txt https://example.com/resource -A "Mozilla/5.0"
+
+# Compliant: save in /dev/null
```

Comment:
One non compliant example comes to my mind:
```
RUN curl -o output.txt https://example.com/resource -X GET
```

Reply:
Should be covered by the test just above at line 419 (unless I'm missing something special about `GET` 🤔):
`RUN curl -o output.txt https://example.com/resource -X POST`